pkg/arvo/sys/zuse.hoon

```diff
@@ -3286,7 +3286,7 @@
     ++  ship                                            ::  string from ship
       |=  a=^ship
       ^-  json
-      (tape (slag 1 (scow %p a)))
+      [%n (rap 3 '"' (scot %p a) '"' ~)]
```

Comment:
json-string nouns are `[%s @t]`, so couldn't this just be `[%s (scot %p a)]`?

But this changes the result, the original discards the leading `~` for whatever reason. So, `[%s (rsh 3^1 (scot %p a))]` perhaps?

Reply:
The follow-up commit (which I should've seen before commenting) addresses the `~`, but the comment about `%s` remains!

Reply:
+en-json's performance is dominated by +jesc -- escaping strings -- and landscape's initial load is mostly ship names, which never need to be escaped.  Using %n lets us skip that while printing.  I got about 1 second better load time with this change.

Reply:
Oh my bad, I missed that in the commit message. Still feels a bit weird, but it does the job for now. Carry on!